### DISCUSSION TITLE
Regex integration?

### DISCUSSION THREAD

**Discussion by epage** [winnow-rs/winnow#67] [Ideas]
While there are likely impedance mismatches, I wonder how helpful it would be to have code using `regex-syntax`, `quote`, and `prettyplease` to code-generate nom output.

Possible uses
- A macro that does snapshot testing to do code-generation
-  CLI to do one-time code generation
- Wasm to support doing translation in the docs.

**Comment by baoyachi** [comment]
Hi @epage ,Is there have any new ideas here?

**Reply by epage** [reply]
Not really.  For myself, I don't find I have needs for regex so I don't have good context for moving something like this forward.
